src/plugins/intel_cpu/src/emitters/jit_snippets_emitters.cpp

```diff
@@ -214,18 +259,19 @@ void TileSchedulerEmitter::validate_arguments(const std::vector<size_t> &in,
         IE_THROW() << "TileSchedulerEmitter can contain only TileEmitters inside its body";
 }
 
-void TileSchedulerEmitter::emit_tiles(const Reg64& reg_inner_amount, const std::vector<Reg64>& data_ptr_regs, size_t vector_size,
+void TileSchedulerEmitter::emit_static_tiles(const Reg64& reg_inner_amount, const std::vector<Reg64>& data_ptr_regs, size_t vector_size,
                                       const std::vector<size_t>& vec_pool, const std::vector<size_t>& gpr_pool) const {
     // TileAllocatedEmitter is just an alias to perform dynamic_pointer_cast only once and reuse it below several times
     using TileAllocatedEmitter = std::pair<std::shared_ptr<TileEmitter>, const ngraph::snippets::RegInfo&>;
     TileAllocatedEmitter vector_tile {std::dynamic_pointer_cast<TileEmitter>(body[0].first), body[0].second};
     TileAllocatedEmitter scalar_tile {std::dynamic_pointer_cast<TileEmitter>(body[1].first), body[1].second};
-    const size_t inner_work_amount = jcp.scheduler_dims[1];
+    const size_t inner_work_amount = jcp.scheduler_work_amounts[1];
     auto process_tile =
         [&](const bool evaluate_once, const TileAllocatedEmitter& tile) {
             // If Tile is evaluated only once, then we can emit its body directly and skip work_amount decrements and checks
             if (evaluate_once) {
                 tile.first->emit_body(vec_pool, gpr_pool);
+                tile.first->emit_ptr_increments_static(data_ptr_regs);
```

Comment:
It's very difficult to understand updates for ptrs incrementing))
Could you please clarify if now we always increment ptrs for case `[1,16]` on AVX512?
In my opinion we shouldn't increment in this case but at the moment seems like we will increment... I'm not sure so could you please verify this moment?

Reply:
Good point, we really need to simplify this logics, as it's already tough to support (and even more difficult to improve)